<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meb_prebook</class>
 <widget class="QDialog" name="meb_prebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelPrebookListText">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>20</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Text*/

position: absolute;
width: 164px;
height: 56px;
left: 112px;
top: 47px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 36px;
line-height: 56px;
display: flex;
align-items: center;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Prebooks List</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="ItemPrebook">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>521</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 8px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGreyBorderPB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>541</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 613px;
height: 104px;
left: 34px;
top: 112px;

background: #C4C4C4;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGreyBorderPB_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>211</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 613px;
height: 104px;
left: 34px;
top: 112px;

background: #C4C4C4;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;
</string>
   </property>
   <property name="text">
    <string>This is a list of members who have requested to prebook a currently un-available book from the library.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;
</string>
   </property>
   <property name="text">
    <string>Member Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbName">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>150</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>180</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;
</string>
   </property>
   <property name="text">
    <string>Member ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMID">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>240</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;
</string>
   </property>
   <property name="text">
    <string>Book Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;
</string>
   </property>
   <property name="text">
    <string>Book ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBID">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>330</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;
</string>
   </property>
   <property name="text">
    <string>Book Due Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDB">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
   </property>
   <property name="text">
    <string>Date Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>470</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;
</string>
   </property>
   <property name="text">
    <string>Member requested prebook on</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRD">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>510</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
   </property>
   <property name="text">
    <string>Requested Date</string>
   </property>
  </widget>
  <zorder>labelGreyBorderPB</zorder>
  <zorder>labelPrebookListText</zorder>
  <zorder>ItemPrebook</zorder>
  <zorder>labelGreyBorderPB_2</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>lbName</zorder>
  <zorder>label_4</zorder>
  <zorder>lbMID</zorder>
  <zorder>label_6</zorder>
  <zorder>lbTitle</zorder>
  <zorder>label_8</zorder>
  <zorder>lbBID</zorder>
  <zorder>label_10</zorder>
  <zorder>lbDB</zorder>
  <zorder>label_12</zorder>
  <zorder>lbRD</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
